src/coreclr/jit/emitxarch.cpp

```diff
@@ -16011,6 +16011,13 @@ emitter::insExecutionCharacteristics emitter::getInsExecutionCharacteristics(ins
             result.insThroughput = PERFSCORE_THROUGHPUT_2X;
             break;
 
+        case INS_pause:
+        {
+            result.insLatency    = PERFSCORE_LATENCY_140C;
```

Comment:
it's 14x lower on pre-Skylakes. It reminds me this issue https://github.com/dotnet/runtime/issues/8744

Reply:
Yes, but all of our latencies are currently dependent on and configured for Skylake.

We'd need some more advanced setup if we wanted to track per architecture latencies/throughput.